### PR TITLE
Create lighthouse.md

### PR DESCRIPTION
**Pull request by nandit123**

### CODE REVIEW

**Comment by MarcusWentz** [comment]
Please update: 

1. img src (I see a page error with):

       npm run start

and 

    npm run build

2. Page title to be "Lighthouse"

File: docs/storage/lighthouse.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Lighthouse Storage and Encryption
+sidebar_position: 4
+---
+
+<img src="https://gateway.lighthouse.storage/ipfs/Qmc5LnkF1QwTst5ikoZyT5yZRzdsuijB5n95eLTDeFNTwb" alt="Lighthouse Logo" height=200px>
```
Comment (MarcusWentz) [inline docs/storage/lighthouse.md:6]: Error on this line

Comment (nandit123) [reply docs/storage/lighthouse.md]: corrected, probably closing tag issue

**Comment by MarcusWentz** [comment]
Please update the following: 

1. Remove logos (to be consistent with other pages).

2. Remove the following section (we have a tutorial like this on another page, so it is repetitive adding it to the Lighthouse page):

https://github.com/Shardeum/shardeum-docs/pull/75/files#diff-3a7b1857f4a83310d48871ba80e315891005e2f9accf156a4fad955fd989f18bR37-R71

3. After these changes are added, please update branch to be synced with the current branch, and then combine all commits to be 1 commit.

**Pull request closed without merging**